tmail-backend/apps/distributed-es6-backport/README.md

```diff
@@ -45,13 +45,13 @@ You can then start in that newly created network all the other softwares the dis
 ```
 docker run -d --network emaily --name=cassandra cassandra:3.11.10
 
-docker run -d --network emaily --name=rabbitmq rabbitmq:3.8.17-management
+docker run -d --network emaily --name=rabbitmq rabbitmq:3.9.18-management
```

Comment:
is there any big change between rabbitmq 3.8 and 3.9? Just wondering

Reply:
No not really.

Reply:
not relate, 
In past, I got an error when running docker rabbit 3.8 on CPU apple silicon chipset. 
While 3.9 works